updated reach section

Dhruv454000 opened the pull request:
- Updated reach section by adding NYU , North eastern uni , TI , AMD , Qualcomm , ARM to our reach section.
- This PR will close: #62

Dhruv454000 commented:
This is how it looks.
![image](https://user-images.githubusercontent.com/84779934/177028838-4908e3a1-59b3-4577-9de2-fc5ed449cc41.png)

gautam-dev-maker commented:
As specified by @OSSome01 we need to differentiate between universities and coporates.

OSSome01 commented:
Add Oregon State Univ. Also, check whether any other universities/corporates are left out.

Dhruv454000 commented:
![image](https://user-images.githubusercontent.com/84779934/180411792-865a4cde-3499-4fae-8483-613b1e2d4c95.png)

OSSome01 commented:
1. Make the rows in the "Our Reach" section symmetric. 
2. It would be better if all the universities and corporates are listed alphabetically.

Dhruv454000 commented:
![image](https://user-images.githubusercontent.com/84779934/180484493-0408dba2-06dc-4364-9b32-f5d111ebfde4.png)

Dhruv454000 commented:
> 1. Make the rows in the "Our Reach" section symmetric.
> 2. It would be better if all the universities and corporates are listed alphabetically.

Alphabetically rename karne ke baad uneven size images sath me aa rhe the acha nhi dikh rha tha corporates me.

Dhruv454000 commented:
![image](https://user-images.githubusercontent.com/84779934/180496535-332639a5-8b62-45b7-8cf7-647b0c4c6681.png)

Dhruv454000 commented:
![image](https://user-images.githubusercontent.com/84779934/183874373-95b5c39b-8424-4854-a9ca-dade8de023b8.png)

- Changed naming of reach section and removed `our` wherever not needed. ex: our universities, our reach, our projects removed our from all this.

VedantParanjape commented:
@Dhruv454000 can you center the images ? Other than that please review the code changes @kkothari2001

kkothari2001 commented on components/OurCorporates/OurCorporates.js:
```diff
@@ -0,0 +1,28 @@
+import { CorporateReachImgNames, CorporateReachLinks } from '../../data';
+import styles from './OurCorporates.module.scss';
+
+const OurCorporates = () => {
+  return (
+    <>
+      <div className={styles.reach}>
+        <h1>Industrial Reach</h1>
+        <div className={styles.reachGroup}>
+          {CorporateReachImgNames.map((name, idx) => {
+            return (
+              <a href={`${CorporateReachLinks[idx]}`}>
+                <img
+                  src={`/static/images/corporates/${name}`}
+                  alt=''
+                  className={styles.reachImg}
+                  key={`alumni_reach_${idx}`}
+                ></img>
+              </a>
+            );
+          })}
+        </div>
+      </div>
+    </>
+  );
+};
+
+export default OurCorporates;
```
Why is a new component needed here? It seems like the difference in the components of Corporate Reach and Our Reach is just the titles and the key prefix. If that's the only difference why not just modify the existing component to accept the title and key prefix as parameters?

Dhruv454000 replied:
> Why is a new component needed here? It seems like the difference in the components of Corporate Reach and Our Reach is just the titles and the key prefix. If that's the only difference why not just modify the existing component to accept the title and key prefix as parameters?

I though corporates and Universities ko completely different rakhna he so I made different components so that easy to identify in future, images ka bhi different folders banaya he.

kkothari2001 replied:
You have a point, but kind of beats the purpose having Components.

Anyway, having seperate component will be better if someone less familiar will try to edit the website in the future (which usually is the case here).

I think it's not wrong to forego a bit of code quality for better future maintenance.

kkothari2001 reviewed (changes requested):
Just a small code change.

But apart from that, just a  few UX opinions:
> The universities or companies should not be in alphabetical order. The ones we want to advertise first should be first. Because as of now, due to the sheer number of logos it becomes difficult to spot a single one. So, if we want to highlight something in particular, that deserves to be in the top line.

> And yes, center align please, space-evenly was already the right choice.

Dhruv454000 commented:
> @Dhruv454000 can you center the images ? Other than that please review the code changes @kkothari2001

yes will do

kkothari2001 reviewed (approved):
LGTM!
